src/Exceptions/HttpUnauthorized.php

```diff
@@ -8,4 +8,3 @@ class HttpUnauthorized extends HttpException {
     public const DESCRIPTION = "Indicates that the request has not been applied because it lacks valid authentication credentials for the target resource.";
     public const HREF = "https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401";
 }
-
```

Comment:
Removed unix line ending - ALWAYS have a unix line ending, it can cause very weird issues.

Just use https://marketplace.visualstudio.com/items?itemName=jkiviluoto.tws you can use this to add them automatically. It also removes trailing whitespaces